aict_tools/scripts/plot_disp_performance.py

```diff
@@ -0,0 +1,126 @@
+import click
+import logging
+import matplotlib
+import matplotlib.pyplot as plt
+import joblib
+import fact.io
+
+from ..configuration import AICTConfig
+from ..plotting import (
+    plot_roc,
+    plot_probabilities,
+    plot_regressor_confusion,
+    plot_bias_resolution,
+    plot_feature_importances,
+    plot_true_delta_delta
+)
+
+if matplotlib.get_backend() == 'pgf':
+    from matplotlib.backends.backend_pgf import PdfPages
+else:
+    from matplotlib.backends.backend_pdf import PdfPages
+
+
+@click.command()
+@click.argument('configuration_path', type=click.Path(exists=True, dir_okay=False))
+@click.argument('performance_path', type=click.Path(exists=True, dir_okay=False))
+@click.argument('data_path', type=click.Path(exists=True, dir_okay=False))
+@click.argument('sign_model_path', type=click.Path(exists=True, dir_okay=False))
+@click.argument('disp_model_path', type=click.Path(exists=True, dir_okay=False))
+@click.option('-o', '--output', type=click.Path(exists=False, dir_okay=False))
+@click.option('-k', '--key', help='HDF5 key for hdf5 for performance_path', default='data')
+@click.option('-k_data', '--key_data', help='HDF5 key for hdf5 for data_path', default='events')
+def main(configuration_path, performance_path, data_path, sign_model_path, disp_model_path, output, key, key_data):
+    ''' Create some performance evaluation plots for the separator '''
+    logging.basicConfig(level=logging.INFO)
+    log = logging.getLogger()
+
+    model_config = AICTConfig.from_yaml(configuration_path).disp
+
+    log.info('Loading perfomance data')
+    df = fact.io.read_data(performance_path, key=key)
+
+    log.info('Loading original data')
+    df_data = fact.io.read_data(data_path, key=key_data, columns = [
+        model_config.source_az_column,
+        model_config.source_zd_column,
+        model_config.pointing_az_column,
+        model_config.pointing_zd_column,
+        model_config.focal_length_column,
```

Comment:
Focal length is not there for FACT, so only load focal length `if config.coordinate_transform == "CTA"`